blocks/rich-text/index.js

```diff
@@ -744,7 +744,8 @@ export class RichText extends Component {
 					if ( ! anchor ) {
 						this.removeFormat( 'link' );
 					}
-					this.applyFormat( 'link', { href: formatValue.value, target: formatValue.target }, anchor );
+					const { value: href, ...params } = formatValue;
+					this.applyFormat( 'link', { href, ...params }, anchor );
```

Comment:
Feels like we could avoid some friction if we named the attribute `href` instead of `value`, but eh.